docs/docs/manual/starting.md

```diff
@@ -80,53 +76,27 @@ This can be useful if you want to pre-select a specific number of rows from your
-
-
-
-*   Add OpenRefine (running from your computer) to an account authorized to access your database
-*   Set up OpenRefine to access that database using that account 
+If you are an administrator or have SQL access to a database of information, you may want to pull the latest dataset directly from there. This could include an online catalogue, a content management system, or a digital repository or collection management system. You can also load a `.db` file saved locally.
 
-![A screenshot of connecting to a database](img/databaseconnect.jpg "A screenshot of connecting to a database.")
+There are some publicly-accessible databases you can query, such as [one provided by Rfam](https://docs.rfam.org/en/latest/database.html). The instructions provided by Rfam can help you understand how to connect to and query from any database. 
 
+OpenRefine can connect to PostgreSQL, MySQL, MariaDB, and SQLite database systems. It will automatically populate the “Port” field based on which of these you choose, but you can manually edit this if needed.
 
-Log in to your hosting provider. Get the database type (such as MySQL), database name, and the URL (either an IP address, such as `127.0.0.1`, or the domain that uses the database, such as _https://openrefine.org/category/blog.html_). Then look at the accounts authorized for access. You may wish to create a new account just for OpenRefine, or add OpenRefine to an existing account. 
-
-Each host will have a slightly different method, but generally speaking: look for “accounts with access” to the database you wish to authorize, and within the settings for that account, look for “allowable hosts” or “access hosts.” 
-
-In that list, add the IP address of your own computer, because that is where the OpenRefine access request will be coming from. You can find this easily, by clicking “Test” within OpenRefine once the rest of the information is filled out: OpenRefine will give you an error that looks like 
-
-``` error:Access denied for user 'yourusername'@'123-45-67-89.yourISP.com' ```
+If you have a downloaded database (`.db`) file from an SQLite database, you can supply the path to the file on your computer directly in the “Database” field at the bottom of the form. You can leave the rest of the fields blank.
 
-Take your IP address from this error message and put that, with periods instead of hyphens, into the “allowable hosts” field on the account you’re trying to use. Add a wildcard to the end of your IP address (“123.45.67.89%”). Save that setting, and then test the connection again with OpenRefine.
+To import data directly from a database online, you will need the database type (such as MySQL), database name, and the hostname (either an IP address, such as `127.0.0.1`, or the domain that uses the database) and the port on the host. You will need an account authorized for access, and you may need to add OpenRefine (connecting from your computer’s IP) to the “allowable hosts” for that account.
```

Comment:
Sorry I'm being picky but:
"the domain that uses the database" I think should be "or the domain that hosts the database"

And at the risk of re-treading previous discussion:
"you may need to add OpenRefine (connecting from your computer’s IP) to the “allowable hosts” for that account." 
I think would be more accurate to say:
"you may need to add the host or IP that OpenRefine is running on to the “allowable hosts” for that account." 

Even this I don't think is entirely accurate - because I think the allowable host will need to be the host/IP that the database perceives OpenRefine to be connecting from - which won't necessarily be the IP that is allocated to the computer running OpenRefine

Reply:
Sorry another picky point - "database online" in the first sentence should be "database" - the question of whether the database is online or not doesn't affect the connection (it could be running locally on the users computer)

Reply:
@ostephens one way to express that is to say "NOTE: Firewall rules and your Database networking rules might require configuration to allow the egress IP of OpenRefine to connect to the ingress IP of your database"

Reply:
I wasn't crazy about "ingress/egress" :) but everything else here is incorporated now.